### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by krishvsoni**
- **What kind of change does this PR introduce?**  - docs update -
-  **Why was this change needed?** (You can also link to an open issue here
- ) Already yaml file of docker is given but it in quick start we need to give Docker official url for downloading according to user device. It will help to setup the docker will also save the time 

- **Other information**: - image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,9 @@
 </p>
 
 <p align="left">
-  <strong><a href="https://docsgpt.arc53.com/">DocsGPT</a></strong> is a cutting-edge open-source solution that streamlines the process of finding information in the project documentation. With its integration of the powerful <strong>GPT</strong> models, developers can easily ask questions about a project and receive accurate answers.
```
Comment (dartpain) [inline README.md:10]: Why are you removing the link?

File: README.md [hunk 2]
```diff
@@ -7,9 +7,9 @@
...
+  <strong>DocsGPT</strong> is a cutting-edge open-source solution that streamlines the process of finding information in project documentation. With its integration of the powerful <strong>GPT</strong> models, developers can easily ask questions about a project and receive accurate answers.
   
-Say goodbye to time-consuming manual searches, and let <strong><a href="https://docsgpt.arc53.com/">DocsGPT</a></strong> help you quickly find the information you need. Try it out and see how it revolutionizes your project documentation experience. Contribute to its development and be a part of the future of AI-powered assistance.
```
Comment (dartpain) [inline README.md:12]: Again

File: README.md [hunk 3]
```diff
@@ -21,62 +21,64 @@ Say goodbye to time-consuming manual searches, and let <strong><a href="https://
  
 </div>
 
-### Production Support / Help for companies:
+### Production Support / Help for companies: 
 
 We're eager to provide personalized assistance when deploying your DocsGPT to a live environment.
-
-- [Book Demo 👋](https://airtable.com/appdeaL0F1qV8Bl2C/shrrJF1Ll7btCJRbP)
```
Comment (dartpain) [inline README.md:28]: Should be book demo, and why are lines removed?

File: README.md [hunk 4]
```diff
@@ -21,62 +21,64 @@ Say goodbye to time-consuming manual searches, and let <strong><a href="https://
...
 ![video-example-of-docs-gpt](https://d3dg1063dc54p9.cloudfront.net/videos/demov3.gif)
 
+
 ## Roadmap
 
 You can find our roadmap [here](https://github.com/orgs/arc53/projects/2). Please don't hesitate to contribute or create issues, it helps us improve DocsGPT!
 
 ## Our Open-Source models optimized for DocsGPT:
 
-| Name                                                                  | Base Model  | Requirements (or similar) |
```
Comment (dartpain) [inline README.md:41]: Table was cleaner, why change it?

File: README.md [hunk 5]
```diff
@@ -21,62 +21,64 @@ Say goodbye to time-consuming manual searches, and let <strong><a href="https://
...
 If you don't have enough resources to run it, you can use bitsnbytes to quantize.
 
+
 ## Features
 
 ![Group 9](https://user-images.githubusercontent.com/17906039/220427472-2644cff4-7666-46a5-819f-fc4a521f63c7.png)
 
-## Useful links
-
-- 🔍🔥 [Live preview](https://docsgpt.arc53.com/)
```
Comment (dartpain) [inline README.md:55]: Why are emojis removed?

File: README.md [hunk 6]
```diff
@@ -87,30 +89,26 @@ It will install all the dependencies and allow you to download the local model o
 Otherwise, refer to this Guide:
 
 1. Download and open this repository with `git clone https://github.com/arc53/DocsGPT.git`
-2. Create a `.env` file in your root directory and set the env variable `OPENAI_API_KEY` with your [OpenAI API key](https://platform.openai.com/account/api-keys) and `VITE_API_STREAMING` to true or false, depending on whether you want streaming answers or not.
```
Comment (dartpain) [inline README.md:90]: Please explain this change?

**Review by dartpain — CHANGES_REQUESTED** [review]
Please check the changes

File: README.md (hunk 6 above)
Comment (krishvsoni) [reply README.md:90]: Wait I fix

**Comment by krishvsoni** [comment]
Done✔️

**Comment by dartpain** [comment]
Can you please link it to instruction on how to install docker not just a documentation website. Thank you!

**Comment by krishvsoni** [comment]
I added install docker url

**Comment by krishvsoni** [comment]
Hello, As we had conflicts before in the branch so I had resolved it, and as mentioned about downloading docker I had put the url in the Docker keyword in the README file.

**Review by dartpain — APPROVED** [review]

**Pull request merged**